azure/http.go

```diff
@@ -0,0 +1,192 @@
+package azure
+
+import (
+	"bytes"
+	"encoding/xml"
+	"fmt"
+	"github.com/MSOpenTech/azure-sdk-for-go/core/http"
+	"github.com/MSOpenTech/azure-sdk-for-go/core/tls"
+)
+
+const (
+	azureManagementDnsName    = "https://management.core.windows.net"
+	msVersionHeader           = "x-ms-version"
+	msVersionHeaderValue      = "2014-05-01"
+	contentHeader             = "Content-Type"
+	defaultContentHeaderValue = "application/xml"
+	requestIdHeader           = "X-Ms-Request-Id"
+)
+
+//sendAzureGetRequest sends a request to the management API using the HTTP GET method
+//and returns the response body or an error.
+func (client *Client) SendAzureGetRequest(url string) ([]byte, error) {
+	if len(url) == 0 {
+		return nil, fmt.Errorf(errParamNotSpecified, "url")
+	}
+
+	response, err := client.sendAzureRequest(url, "GET", "", nil)
+	if err != nil {
+		return nil, err
+	}
+
+	responseContent := getResponseBody(response)
+	return responseContent, nil
+}
+
+//sendAzurePostRequest sends a request to the management API using the HTTP POST method
+//and returns the response body or an error.
```

Comment:
It is a small thing, but method returns requestId or an error. Same in SendAzurePutRequest and SendAzureDeleteRequest